code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -359,6 +359,9 @@
 	return ..() | update_flags
 
 /datum/reagent/medicine/omnizine/overdose_process(mob/living/M, severity)
+	if(HAS_TRAIT(M, TRAIT_BADASS))
+		return
```

Comment:
The PR only mentions an immunity to nicotine addiction. Could you either update it to also mention Omnizine not having an OD effect?

Reply:
I intended to not put every effect in the description so as to make it more rewarding to experiment (or codedive) but I should probably add that effect yeah. I'll edit the PR desc